packages/react-native-sdk/src/components/Participant/ParticipantView/ParticipantLabel.tsx

```diff
@@ -26,17 +23,15 @@ export const ParticipantLabel = ({
   participant,
   videoMode,
 }: ParticipantLabelProps) => {
-  const { name, userId, pin, sessionId, isLocalParticipant } = participant;
+  const { name, userId, pin, publishedTracks, sessionId, isLocalParticipant } =
+    participant;
   const call = useCall();
-  const { useCameraState, useMicrophoneState } = useCallStateHooks();
-  const { status: micStatus } = useMicrophoneState();
-  const { status: cameraStatus } = useCameraState();
   const { t } = useI18n();
   const participantName = name ?? userId;
   const participantLabel = isLocalParticipant ? t('You') : participantName;
   const isPinningEnabled = pin?.isLocalPin;
-  const isAudioMuted = micStatus === 'disabled';
-  const isVideoMuted = cameraStatus === 'disabled';
+  const isAudioMuted = !publishedTracks.includes(SfuModels.TrackType.AUDIO);
+  const isVideoMuted = !publishedTracks.includes(SfuModels.TrackType.VIDEO);
```

Comment:
This should be supported in the future, I think the only missing piece is to update camera/mic status when we are muted remotely, it's on my todo list

Reply:
awesome, thanks